### PR TITLE
Create a GET products method

### PR DESCRIPTION
**Pull request by pinocchiochang**
This method will get the info about all the products

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -19,6 +19,24 @@ def index():
...
+def get_product_list():
+  """ Get info about all products
+
+    This method will get the info about all the products
+
+    Args:
+      no arguments
+
+    Returns:
+      response: product information(product id, location id, used/new/open_box, total_quantity, restock_level)
```
Comment (p2hang) [inline server.py:32]: Nice details!
Maybe add returns for some other cases?

**Review by p2hang — APPROVED** [review]
Looks good!

**Review by BobZcao — APPROVED** [review]
GOOD

File: server.py (hunk 1 above)
Comment (pinocchiochang) [reply server.py:32]: Like? @plzhang

**Pull request merged**